lib/src/main/kotlin/com/swmansion/starknet/account/StandardAccount.kt

```diff
@@ -541,7 +541,10 @@ class StandardAccount @JvmOverloads constructor(
     private fun buildEstimateFeeV3Payload(calls: List<Call>, nonce: Felt): List<ExecutableTransaction> {
         val executionParams = InvokeParamsV3(
             nonce = nonce,
-            l1ResourceBounds = ResourceBounds.ZERO,
+            resourceBounds = ResourceBoundsMapping(
+                ResourceBounds.ZERO,
+                ResourceBounds.ZERO,
+            ),
```

Comment:
Nit:

Should we introduce `ResourceBoundsMapping.ZERO`? 🤔

Reply:
I was also thinking about this, should be handy 👍 .